### PR TITLE
Latest GL-C-006P normal version firmware

### PR DESCRIPTION
**Pull request by mariustt**
Related: https://github.com/Koenkk/zigbee-OTA/issues/483

### CODE REVIEW

**Comment by Koenkk** [comment]
How do we make sure this firmware does not end up on the mini? Does someone have the data/database.db entry of the mini ?

**Comment by mariustt** [comment]
I received this OTA from the manufacturer. 
But for now let's leave this PR for now, I'm still trying different options since updating to the latest version and solving a problem with one of my led driver after updating the mini OTA. It still does not work as described in this issue - #483.

Once I have found a solution and tested it I will share the final solution. In any case, should probably think about a solution to differentiate between mini and normal entities in z2m.

**Comment by mariustt** [comment]
@Koenkk I confirm that GL-C-006P_V20551203_KEY_20240227.ota is indeed working and is the latest version.
This OTA version has solved the problem not only for me: 
- https://github.com/Koenkk/zigbee2mqtt/issues/22287 
- https://github.com/Koenkk/zigbee-OTA/issues/463 
- https://github.com/Koenkk/zigbee2mqtt/issues/22287

I just want to point out that this OTA is for normal devices only. If it is used with mini devices, it is likely that they will not work.

File: index.json [hunk 1]
```diff
@@ -3555,5 +3555,14 @@
         "sha512": "178498b7ad89b81792ad34df675167fcbee3c306e6bbe7336b97fe06f92b5e919d6c6b9a14788172e47deaa4422b977fed38fce7af61dd92a471886325421cc7",
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Innr/innr_controlleddevice-sp244-1.6.22.ota",
         "path": "images/Innr/innr_controlleddevice-sp244-1.6.22.ota"
+    },
+    {
```
[the remark this answers is not in the document]
Comment (mariustt) [reply index.json:3559]: Sorry I forgot to add the modelId when updating the image. I've updated it, please check if everything is OK now.

**Comment by Koenkk** [comment]
Thanks!

**Comment by Koenkk** [comment]
Reverted this because of https://github.com/Koenkk/zigbee2mqtt/issues/23128

**Pull request merged**